xAPI.md

```diff
@@ -544,9 +552,22 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 <a name="xapi-components" />
 
-## 5.0 xAPI Components
+<a name="xapi_ecosystem"/>
+## 5.0 The xAPI Ecosystem
```

Comment:
I think, given the existing content of this section, that I prefer 'xAPI Components' . I don't think Andy's second two diagrams fit the 'ecosystem' title.